src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -607,50 +741,44 @@ public static async ValueTask<FormulaValue> AverageTable(EvalVisitor runner, Eva
                 return CommonErrors.DivByZeroError(irContext);
             }
 
-            return await RunAggregatorAsync("Average", new AverageAgg(), runner, context, irContext, args).ConfigureAwait(false);

```

Comment:
>ConfigureAwait [](http://example.com/codeflow?start=107&length=14)

keep ConfigureAwait

Reply:
That line is old enough that it predates the addition.  Added back in.